src/gamatrix/__main__.py

```diff
@@ -117,7 +117,7 @@ def upload_file():
         <!doctype html>
         <title>Upload DB</title>
         <h1>Upload DB</h1>
-        GOG DBs are usually in C:\ProgramData\GOG.com\Galaxy\storage\galaxy-2.0.db
+        GOG DBs are usually in C:\\ProgramData\\GOG.com\\Galaxy\\storage\\galaxy-2.0.db
```

Comment:
Why the double slashes?

Reply:
Warnings in a run of `mypy` IIRC.

Reply:
No I was mistaken. The new Intellisense engine for VSCode-Python was the source of the warning. `Pylance`.

![image](https://user-images.githubusercontent.com/34773432/201578610-3054f3a1-643e-4f02-827e-0ec9765a20e3.png)

Reply:
(the other two are due to use of `or` in an atomic non-evaluated line of code)

Reply:
Ok, we _could_ tell Pylance to ignore the line since it's not Python. But this is fine; I checked and it still renders with single backslashes so no harm.

Reply:
I tried to use `# type: ignore` to get around this (and tell `Pylance` to shush here) but apparently it didn't respect the multiline string layout (I have to separate out the one line with backslashes, gross). So I'm going with double-backslash. Let's all just blame Microsoft for this one all up and be done with it.